src/api/app/models/cloud/backend/upload_job.rb

```diff
@@ -18,8 +18,16 @@ class UploadJob
                      :repository,
                      :arch,
                      :filename,
+                     :size,
                      :vpc_subnet_id,
-                     :size
+                     :image_name,
```

Comment:
This coming directly from the backend, it does not know about these information (and you also don't need it to display, right?)
(You can drop the vpc_subnet_id as well)

Reply:
Those are needed for the **text_field** inputs in the view, in the end the object must respond to the fields.

Reply:
I know why you did it but it is wrong because these fields do not exist on the backend job! If you fetch a backend job, these fields will all be nil!

You can create form fields also like this:

```
= text_field('cloud_backend_upload_job', 'image_name')
```

(and yes it is wrong for the vpc subnet as well)

Reply:
Ok, will do